src/components/Book.js

```diff
@@ -1,29 +1,85 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
+import { CircularProgressbar } from 'react-circular-progressbar';
 import { removeBook, deleteBook } from '../redux/features/books/booksSlice';
 
+import 'react-circular-progressbar/dist/styles.css';
+
 function Book({
   itemId, title, author, category,
 }) {
+  const [chapterNumber, setChapterNumber] = useState({
+    chapter: 0,
+    percent: 0,
+  });
   const dispatch = useDispatch();
 
   const remover = (itemId) => {
     dispatch(deleteBook(itemId)).then(() => dispatch(removeBook(itemId)));
   };
   return (
-    <div>
-      <p>{category}</p>
-      <h2>{title}</h2>
-      <p>{author}</p>
+    <div className="book-container">
+      <div className="book-item-main">
+        <p className="category">{category}</p>
+        <h2>{title}</h2>
+        <p className="author">{author}</p>
+        <button type="submit" className="button-show">
+          Comments
+        </button>
+        <button
+          type="submit"
+          className="button-show remove"
+          onClick={() => remover(itemId)}
+          z
+        >
+          Remove
+        </button>
+        <button type="submit" className="button-show">
+          Edit
+        </button>
+      </div>
```

Comment:
- [ ] As I mentioned above :point_up_2: please follow [Zaplin](https://zeplin.io/) design but in this portion, you also do not follow the font styles and colours so kindly use the same fonts and styles mentioned in [Zaplin](https://zeplin.io/) design :+1: check below :point_down:  the comparison between your design and [Zaplin](https://zeplin.io/) design :slightly_smiling_face: 

| Your Design | Zaplin Design|
|--------|--------|
| ![Screenshot from 2023-05-13 16-14-56](https://github.com/Lul-Abdifan/Bookstore/assets/45226848/a0a9a8aa-5420-4389-b9e7-203c908c8895) | ![Screenshot from 2023-05-13 16-15-12](https://github.com/Lul-Abdifan/Bookstore/assets/45226848/c359e0f5-7740-444b-9e89-e25baf2521d1) |